### ISSUE TITLE
Labelling Module Development

### ISSUE THREAD

**Issue by kunnark** [buerokratt/Data-Anonymizer#44]

**Comment by turnerrainer** [comment]
@kunnark user story and AC title does not correspond to the style we have agreed upon.

**Comment by kunnark** [comment]
@turnerrainer this is an epic and epic is not a user story in essence. Should I format this as user story instead?

**Comment by turnerrainer** [comment]
> @turnerrainer this is an epic and epic is not a user story in essence. Should I format this as user story instead?

Yes, let's keep the same style throughout all issues (and issues marked and treated as epics).
